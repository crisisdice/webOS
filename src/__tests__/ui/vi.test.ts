import { runAll } from '../ui'
import { vi } from '../cases'

runAll(vi)
