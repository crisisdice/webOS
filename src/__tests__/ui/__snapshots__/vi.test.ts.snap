// Vitest Snapshot v1

exports[`vi<ENTER>ihello 1`] = `"<body><div><div><div><div class=\\"buffer svelte-1cykpto\\"> <div class=\\"line svelte-1cykpto\\">1 <span>hello<span class=\\"caret-line svelte-136ugrd\\">ㅤ</span></span></div> </div> <div class=\\"footer svelte-1cykpto\\"> <span class=\\"coords svelte-1cykpto\\">1,6</span></div></div> <input autofocus=\\"\\"></div></div></body>"`;

exports[`vi<ENTER>ihello<ENTER>hello worldArrowUp 1`] = `"<body><div><div><div><div class=\\"buffer svelte-1cykpto\\"> <div class=\\"line svelte-1cykpto\\">1 <span>hello<span class=\\"caret-line svelte-136ugrd\\">ㅤ</span></span></div> <div class=\\"line svelte-1cykpto\\">2 hello world</div></div> <div class=\\"footer svelte-1cykpto\\"> <span class=\\"coords svelte-1cykpto\\">1,6</span></div></div> <input autofocus=\\"\\"></div></div></body>"`;

exports[`vi<ENTER>ihello<LEFT 6><RIGHT 6> 1`] = `"<body><div><div><div><div class=\\"buffer svelte-1cykpto\\"> <div class=\\"line svelte-1cykpto\\">1 <span>hello<span class=\\"caret-line svelte-136ugrd\\">ㅤ</span></span></div> </div> <div class=\\"footer svelte-1cykpto\\"> <span class=\\"coords svelte-1cykpto\\">1,6</span></div></div> <input autofocus=\\"\\"></div></div></body>"`;

exports[`vi<ENTER>ihello\` 1`] = `"<body><div><div><div><div class=\\"buffer svelte-1cykpto\\"> <div class=\\"line svelte-1cykpto\\">1 <span>hell<span class=\\"caret-full svelte-136ugrd\\">o</span></span></div> </div> <div class=\\"footer svelte-1cykpto\\"> <span class=\\"coords svelte-1cykpto\\">1,5</span></div></div> <input autofocus=\\"\\"></div></div></body>"`;

exports[`vi<ENTER>ihello\`:cmd 1`] = `"<body><div><div><div><div class=\\"buffer svelte-1cykpto\\"> <div class=\\"line svelte-1cykpto\\">1 hello</div> </div> <div class=\\"footer svelte-1cykpto\\">: <span>cmd<span class=\\"caret-empty svelte-136ugrd\\">_</span></span> <span class=\\"coords svelte-1cykpto\\">1,5</span></div></div> <input autofocus=\\"\\"></div></div></body>"`;

exports[`vi<ENTER>ihello\`:cmd<LEFT 6><RIGHT 6> 1`] = `"<body><div><div><div><div class=\\"buffer svelte-1cykpto\\"> <div class=\\"line svelte-1cykpto\\">1 hello</div> </div> <div class=\\"footer svelte-1cykpto\\">: <span>cmd<span class=\\"caret-empty svelte-136ugrd\\">_</span></span> <span class=\\"coords svelte-1cykpto\\">1,5</span></div></div> <input autofocus=\\"\\"></div></div></body>"`;
